loader/loader.go

```diff
@@ -177,6 +178,39 @@ func (l *loader) GetRelease(ctx context.Context, cli client.Client, name, namesp
 	return release, toolkit.GetObject(name, namespace, cli, ctx, release)
 }
 
+// GetPreviousRelease returns the Release that was created just before the given Release.
+// If no previous Release is found, it returns a NotFound error.
+func (l *loader) GetPreviousRelease(ctx context.Context, cli client.Client, release *v1alpha1.Release) (*v1alpha1.Release, error) {
+	releases := &v1alpha1.ReleaseList{}
+	err := cli.List(ctx, releases, client.InNamespace(release.Namespace))
+	if err != nil {
+		return nil, err
+	}
+
+	var previousRelease *v1alpha1.Release
+	releaseTime := release.CreationTimestamp.Time
+
+	fmt.Printf("Current release: %s, Timestamp: %v\n", release.Name, releaseTime)
```

Comment:
We don't want prints like this.

Reply:
Yes, ignore `Printf` for now. I added those just for some debugging and logs while testing.